external/uapaotredist/project.json.template

```diff
@@ -0,0 +1,15 @@
+﻿{
+  "frameworks": {
+    "uap10.1": {
+      "dependencies": {
+        "System.Diagnostics.Tracing": "4.4.0-beta-24906-01",
```

Comment:
This is the exact version of the package that the ProjectN ILC Toolchain uses. I basically went and restored the project.json that PN uses, and checked the exact version for each of the libraries that needed to be redisted, and they are using this beta version for Tracing.